src/test/java/com/microsoft/sqlserver/testframework/DBTable.java

```diff
@@ -108,7 +131,7 @@ private void addColumns() {
         columns = new ArrayList<>(totalColumns);
 
         // Add RowID column
-        SqlType sqlType = schema.getSqlType(1); // Type SqlInt
+        SqlType sqlType = new SqlInt(); // Type SqlInt
```

Comment:
Why is this changed here?

Reply:
getSqlType(1) only works if one of the columns is int before adding the index column, therefore it fails when making a test which does not have to have an int column.

Reply:
If that works before, we keep it working as it is. The newly added test should not break the existing logic.

Reply:
Does the existing logic work not because of good coding, but because the existing test cases happen to meet the broken logic's requirements?

Reply:
The initial logic is not broken if you read the code flow where the `addColumns()` is called. The `sqlTypes.add(new SqlInt());` is executed before when `autoGenerateSchema` is true. And `addColumns()` and `addColumns(boolean)` are only called when  `autoGenerateSchema` is true. If this needs to be changed to `new SqlInt()`, you will need to change both places to keep the consistency.